server/index.js

```diff
@@ -1,30 +1,129 @@
+/* global require */
 const express = require('express');
 const app = express();
-const {getListing} = require('../database/index.js');
+const {getListing, Listing} = require('../database/index.js');
 const cors = require('cors');
-
+const bodyParser = require('body-parser');
 app.use(cors());
 
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: true}));
+// eslint-disable-next-line no-undef
 app.use(express.static(__dirname + '/../client/public/dist'));
 
-app.get('/api/intro/:id', function (req, res) {
-  var id = req.params.id;
+app.get('/api/intro/:listingId', function (req, res) {
+  var id = req.params.listingId;
   getListing(id, function(err, result) {
     if (err) {
-      console.log('fail to get intro')
+      console.log('fail to get intro');
     } else {
-      res.send(result)
+      res.send(result);
     }
   });
 });
 
-app.get('/app.js', cors(), function (req, res) {
-  res.sendFile(path.join(__dirname, '../client/public/dist/app.js'))
+// POST Request - Adds a record to
+app.post('/api/intro/', (req, res) => {
+  /**
+   * Schema for db.listings():
+   * {
+   *    listingNumber: Number,
+   *    photos: array(5 to 6, urlStrings), // 0 - 6.
+   *    title: String,
+   *    description: String
+   * }
+   *
+   */
+
+  Listing.create(req.body)
+    .then((result) => {
+      res.send(result);
+      res.status(200);
+    })
+    .catch((err) => {
+      res.send(err);
+      res.status(500);
+    });
```

Comment:
Nice and clean.
Question - is the listing id getting passed in with the rest of the data in the body, or are you generating that elsewhere?
Suggestion - a good refactor of this would be to get the database piece of this logic over in database/index.js; it will make changing the db later easier, and make this file easier to test.
Also - you may want to remove sending the whole error message to the client; use a console.error instead. You could still send a shorter failure response that doesn't have as much deatil.